xopt/bayesian/optimize.py

```diff
@@ -162,12 +164,17 @@ def save(pop, prefix, generation):
         # horiz. stack objective and constraint results for training/acq specification
         train_outputs = torch.hstack((standardized_train_y, corrected_train_c))
 
+        # create and train model
+        model_start = time.time()
         model = create_model(train_x, train_outputs, input_normalize, custom_model)
+        if verbose:
```

Comment:
`vprint` has this logic, you don't need additional `if verbose`. Also consider putting logger messages in the definition of `vprint`.